.circleci/config.yml

```diff
@@ -56,7 +63,13 @@ jobs:
           - checkout
           - run:
                 name: Build classes to scan
-                command: ./gradlew build -x test
+                command: |
+                    ./gradlew build -x test
+                    mkdir -p TEMPORARYclassfiles
+                    cp -r */build/classes/java/main/* TEMPORARYclassfiles/
+                    java -jar jacocoJars/jacococli.jar report jacocoOutput/jacoco-it.exec --classfiles TEMPORARYclassfiles/ --xml ./jacocoOutput/jacocoXMLReport
```

Comment:
need to replace -checkout with attach_workspace at line 63 to carry over files from build step (you can see steps that are failing on the conversation page, the link will take you to circleci page that shows more details as it runs)